cluster/addons/fluentd-gcp/scaler-policy.yaml

```diff
@@ -8,6 +8,12 @@ spec:
   group: scalingpolicy.kope.io
   versions:
     - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true
```

Comment:
while `x-kubernetes-preserve-unknown-fields: true` is not recommended in general when transitioning to v1, this addon is not expected to be used long-term, and this matches current behavior

Reply:
In general we expect that fluentd-gcp-scaler can be removed in one-two month. We are in the process of migrating our last customers from it. So in the worst case it will be removed in 1.23 maybe in 1.22.